packages/duppy/duppy.0.7.1/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-duppy"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "duppy"]
+depends: [
+  "camlp4"
```

Comment:
Does it require the runtime part of camlp4?

Reply:
Yes but it's actually an optional dependency.. Fixing..